include/hal/library/cryptlib/cryptlib_cert.h

```diff
@@ -80,6 +80,23 @@ extern bool libspdm_x509_get_serial_number(const uint8_t *cert, size_t cert_size
                                            uint8_t *serial_number,
                                            size_t *serial_number_size);
 
+/**
```

Comment:
Need `#if LIBSPDM_ADDITIONAL_CHECK_CERT`.

Reply:
Thanks. I have added the API under the MACRO.